pyth-sdk-cw/Cargo.toml

```diff
@@ -10,11 +10,14 @@ description = "pyth price oracle data structures and example usage"
 keywords = [ "pyth", "cw", "cosmwasm", "oracle" ]
 readme = "README.md"
 
+[features]
+test-utils = []
```

Comment:
why we need a feature flag for this? 🤔 I think it is easier for protocols to use it without feature flag (and it does not have any extra dependency) or maybe have it only for test?

Reply:
i searched around for the best way to include test utilities in a crate and this seemed to be the best practice. If the test logic were all under `cfg(test)`, it's not accessible from other crates. This is quite easy to use in tests (you add the line to devDependencies per the Cargo.toml example above).

Reply:
Hmmm I still don't agree much. I mean what is wrong with having a submodule named `testing` without any feature flag?

Just like this:
https://docs.rs/cosmwasm-std/latest/cosmwasm_std/testing/index.html

Reply:
but anyway i approve this to not block you. i'm not an expert here 😅

Reply:
I do think It's a bit brittle also, I think it would be cleaner to either have it not be feature flagged or be a whole separate crate. Solana does a separate crate for example, and both cosmwasm and NEAR both just have freestanding modules as Ali said. The freestanding module sounds the most pleasant for users.